src/style.css

```diff
@@ -0,0 +1,4 @@
+@import url(./blocks/reset.css);
+@import url(./blocks/product-card.css);
+@import url(./blocks/stars.css);
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
```

Comment:
You have already added fonts to your html file. You don't have to do it one more time here.